MyServlet/src/main/webapp/Login/LoginCheck.java

```diff
@@ -0,0 +1,7 @@
+package Login;
+
+public class LoginCheck {
+    public static boolean checkLog (String Name, String Password) {
+        return Name.equals("1") && Password.equals("qwerty");
```

Comment:
The better way to use the following comparison "totally not null string".equals(name)  for avoiding NullPointerException in case of null input.
And also according to name convention plz rename vars to `name` and `password` with a lowercase first letter.
https://www.javatpoint.com/java-naming-conventions

Reply:
And if we use `username` in loginForm.jsp lets continue this naming and in java classes